Changes

```diff
@@ -222,15 +222,21 @@ Working version
   (Antonin Décimo, Damien Doligez, review by Sébastien Hinderer,
    Damien Doligez, Gabriel Scherer, and Xavier Leroy)
 
+* #12497, #12613: Make ocamlc/ocamlopt fail with an error when no
+  input files are specified to build an executable.
+  (Antonin Décimo, review by Sébastien Hinderer)
+
```

Comment:
This diff seems unrelated.

Reply:
Indeed - that was lazy of me to leave it here! I pushed that directly in 8ec2b3dfd488a81a0cef09106cbcd7325c25ebbb.